blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/formats/MetricValidator.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2016 Rackspace
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.rackspacecloud.blueflood.inputs.formats;
+
+import com.rackspacecloud.blueflood.service.Configuration;
+import com.rackspacecloud.blueflood.service.CoreConfig;
+import org.apache.commons.lang.StringUtils;
+
+public class MetricValidator {
```

Comment:
I am not sure I like this class. It seems to be doing something very specific, very narrow, and yet the name is so generic. It will be come a dumping ground of unrelated things. It also has static methods. Do you really need to make this class? Why can't we keep the methods in the class that calls them?